### PR TITLE
Front+Backend

### PR DESCRIPTION
**Pull request by himanshk96**

### CODE REVIEW

File: app/src/components/ProjectFilters.vue [hunk 1]
```diff
@@ -35,6 +35,15 @@
           />
         </div>
 
+        <div class="section">
+          <p class="font-display font-medium text-sm mb-2 px-2">Expertise Level</p>
+          <CheckboxGroup
+            prefix="expertise_level"
+            :keys="['Beginner', 'Intermediate', 'Advanced']"
+            :values="['beginner', 'intermediate', 'advanced']"
```
Comment (dumbbillyhardy) [inline app/src/components/ProjectFilters.vue:43]: I did all caps for these, we'll probably want to do that here too

File: app/src/views/Product.vue [hunk 2]
```diff
@@ -409,6 +432,14 @@ export default class Product extends Vue {
     );
   }
 
+  get showBeginer(): boolean {
+    console.log('a',this.showAllTypes)
```
Comment (dumbbillyhardy) [inline app/src/views/Product.vue:436]: I don't think we need this log

File: app/src/views/Product.vue [hunk 3]
```diff
@@ -409,6 +432,14 @@ export default class Product extends Vue {
     );
   }
 
+  get showBeginer(): boolean {
```
Comment (dumbbillyhardy) [inline app/src/views/Product.vue:435]: Is this used anywhere?

File: app/src/views/Product.vue [hunk 4]
```diff
@@ -287,12 +298,13 @@ export default class Product extends Vue {
...
-    const noneChecked = this.filters.categories.every((c) => !c.checked);
-    if (noneChecked) {
+    const noneCategoryChecked = this.filters.categories.every((c) => !c.checked);
+    const noneExpertiseChecked = this.filters.expertise_level.every((c) => !c.checked);
+    if (noneCategoryChecked && noneExpertiseChecked) {
       return null;
     }
 
-    return this.filters.categories.filter((x) => x.checked).map((x) => x.value);
+    return this.filters.categories.filter((x) => x.checked).map((x) => x.value).concat(this.filters.expertise_level.filter((x) => x.checked).map((x) => x.value));
```
Comment (dumbbillyhardy) [inline app/src/views/Product.vue:307]: `this.filters.expertise_level.filter((x) => x.checked).map((x) => x.value)`
This actually needs to go somewhere else. In queryParams(), we need to add to the where attribute and tell it to look at "metadata.expertise"

something like the following
```
  const expertises = this.filters.expertise_level.filter((x) => x.checked).map((x) => x.value);
  if (chosenExpertises.length > 0) {
      q.where = [...q.where,
        {
          fieldPath: "metadata.expertise",
          operator: "array-contains-any",
          value: chosenExpertises,
        },
      ];
    }
 ```
 
 That doesn't handle when tags aren't queried as well, I think, so you'll want to handle the nulls and stuff as well. (maybe default q.where to empty array? that would do it)

File: app/src/components/ProjectFilters.vue (hunk 1 above)
Comment (himanshk96) [reply app/src/components/ProjectFilters.vue:43]: Acknowledged.

File: app/src/views/Product.vue (hunk 2 above)
Comment (himanshk96) [reply app/src/views/Product.vue:436]: Acknowledged.

File: app/src/views/Product.vue (hunk 3 above)
Comment (himanshk96) [reply app/src/views/Product.vue:435]: I will remove it. Thank you!

File: app/src/views/Product.vue (hunk 4 above)
Comment (himanshk96) [reply app/src/views/Product.vue]: Hey billy, I tried this approach of  ``` q.where = [...q.where, ``` 
It deosnt work. I also couldn't find anyother approach to adding 2 fieldpath into a single query. 

Can you help here? 
The last option I can think of is to create a separate @watcher method to do a 2nd query on 'expertise' feildPath and leave the current @watcher method as it is for 'tags'.

**Pull request closed without merging**